### DISCUSSION TITLE
Where the Can bus sniffing program ?

### DISCUSSION THREAD

**Discussion by Yamh631** [cedricp/ddt4all#952] [Q&A] [answered]
Hello,

How to launch the Can bus sniffing program ?

Thanks

![Capture d'écran 2023-11-26 232039](https://github.com/cedricp/ddt4all/assets/35656304/cfe3dc7d-07d7-415a-9649-0ab2626c8a32)

**Comment by KarelSvo** [comment, marked as answer]
https://youtu.be/SjDC7fUMWmg?t=86

**Reply by Yamh631** [reply]
Thanks!

**Comment by monalos** [comment]
Hello, can I get the can bus information when the buttons on the remote are pressed? I need to know what ID and data it sends when I press the unlock button.
